### PR TITLE
Added metadata to colors.py

### PR DESCRIPTION
**Pull request by Aman-Jamshed**
- [x] issues: fixes #11765 

Added metadata to colors.py in examples/models/colors.py

### CODE REVIEW

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
This LGTM but @tcmetzger can also take a look over towards the end of this week

File: examples/models/colors.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
+'''This example demonstrates the usage of grouping different shades
+of same color together using bokeh groups.
```
Comment (tcmetzger) [inline examples/models/colors.py]:
```suggestion
of the same color together using Bokeh groups.
```
Please capitalize Bokeh (https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#commonly-used-terms)

**Review by tcmetzger — CHANGES_REQUESTED** [review]
Thank you, just two very small requests!

**Review by tcmetzger — APPROVED** [review]
Thank you!

**Pull request merged**